.github/workflows/cifuzz.yml

```
@@ -0,0 +1,26 @@
+name: CIFuzz
+on:
+  pull_request:
+    - master
```

I guess this will not start CIFuzz for commits that had been directly pushed to the upstream.
Consider using something like this

```suggestion
on:
  pull_request:
    types:
      - synchronize
      - reopened
      - opened
    branches:
      - master
    push:
  push:
    branches:
      - master
```